core/src/avm2/globals/flash/display/Shader.as

```diff
@@ -4,7 +4,7 @@ package flash.display {
     import __ruffle__.stub_setter;
 
     public class Shader {
-        private var _data:ShaderData;
+        private var _data:ShaderData = new ShaderData(new ByteArray());
```

Comment:
I don't think this should be initialized- in Flash, `new Shader().data` returns `null`.